packages/devtools_app/lib/src/shared/preferences.dart

```diff
@@ -38,6 +41,8 @@ class PreferencesController {
     });
 
     setGlobal(PreferencesController, this);
+
+    await _inspectorPreferences.init();
```

Comment:
nit, make setting the global the last line in this method. just so we know everything has been initialized before we set the global.